cmd/restic/cmd_tag.go

```diff
@@ -58,7 +59,12 @@ func init() {
 	initMultiSnapshotFilter(tagFlags, &tagOptions.SnapshotFilter, true)
 }
 
-func changeTags(ctx context.Context, repo *repository.Repository, sn *restic.Snapshot, setTags, addTags, removeTags []string) (bool, error) {
+type changedSnapshot struct {
+	Original restic.ID `json:"original"`
```

Comment:
The field name doesn't specify that this is a snapshot id.

Reply:
Done. Added SnapshotID to the name,